src/features/content-insights/charts/bar/__tests__/Bar.test.tsx

```diff
@@ -70,5 +84,48 @@ describe('features/content-insights/charts/bar/Bar', () => {
 
             expect(onMouseLeave).toBeCalled();
         });
+
+        test('should return the bar offset onMouseEnter', () => {
+            const elementOffset = {
+                bottom: 0,
+                height: 50,
+                left: 50,
+                right: 0,
+                top: 20,
+                toJSON: jest.fn(),
```

Comment:
what's `toJSON` from? is this needed?

Reply:
Hi @tjuanitas. I added because it was giving some typescript errors because the prop was missing. I updated it to make it consistent on how we are mocking the DOMRect on the repo